packages/icons/src/stories/trimet-mode-icon-2021.story.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import TrimetModeIcon from "../trimet-mode-icon-2021";
+
+import ModeIconRenderer from "./mode-icon-renderer";
+
+export default {
+  title: "Icons/TrimetModeIcon2021",
+  component: TrimetModeIcon
+};
+
+export const TrimetModeIconExamples = () => {
+  return <ModeIconRenderer component={TrimetModeIcon} />;
```

Comment:
`return` notation is not necessary.

Reply:
Removed in ff496dcc6ea62056eee98d763c9e14f1c49e0d98